plugins/modules/api_info.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: api_info
+
+author:
+  - Manca Bizjak (@mancabizjak)
+  - Miha Dolinar (@mdolin)
+  - Tadej Borovsak (@tadeboro)
+  - Matej Pevec (@mysteriouswolf)
+short_description: Manage ServiceNow GET requests
+description:
+  - Retrieve information about ServiceNow REST Table API results from arbitrary table
+  - For more information, refer to the ServiceNow REST TAble API documentation at
+    U(https://docs.servicenow.com/bundle/paris-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPIO).
+version_added: 1.4.0
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.sys_id.info
+seealso:
+  - module: servicenow.itsm.api
+options:
+  resource:
+    description:
+      - The name of the table that we want to obtain records from
+    type: str
+    required: true
+  sysparm_query:
```

Comment:
or eventually, maybe better to keep it as `sysparm_query`, to distinguish it from the `query` parameter used in other info modules.

Reply:
Yep. Had some problem writing documentation if this attribute was called just `query`, since another query is already defined as dict in method `get_spec` here: https://github.com/ansible-collections/servicenow.itsm/blob/feb3d161cdb1d5052998db569224760a3da26032/plugins/module_utils/arguments.py#L129 so I renamed it to `sysparm_query`. I this it's refered as sysparm_query in other modules as well, for examle here: https://github.com/ansible-collections/servicenow.itsm/blob/feb3d161cdb1d5052998db569224760a3da26032/plugins/inventory/now.py#L425

Reply:
with @uscinski we agreed to leave this variable as it is now since it would have a chance to lead to misunderstanding as `query` is used elsewhere in a different meaning.